tests/common/member_test.go

```diff
@@ -52,3 +55,157 @@ func TestMemberList(t *testing.T) {
 		})
 	}
 }
+
+func TestMemberAdd(t *testing.T) {
+	testRunner.BeforeTest(t)
+
+	tcs := []struct {
+		name    string
+		learner bool
+	}{
+		{
+			name:    "NotLearner",
+			learner: false,
+		},
+		{
+			name:    "Learner",
+			learner: true,
+		},
+	}
+
+	for _, tc := range tcs {
+		for _, nc := range clusterTestCases {
+			t.Run(tc.name+"/"+nc.name, func(t *testing.T) {
+				ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+				defer cancel()
+				clus := testRunner.NewCluster(ctx, t, nc.config)
+				defer clus.Close()
+				cc := clus.Client()
+
+				testutils.ExecuteUntil(ctx, t, func() {
+					clus.WaitLeader(t)
```

Comment:
Do we need `WaitLeader` here? I haven't seen it used in other tests as I would expect that `NewCluster` returns me cluster after leader is established.

Reply:
No. It turns out we don't need `WaitLeader` here. I'll remove them.

The first time I saw `etcdserver: unhealthy cluster` error after calling `MemberAdd`, I thought it was because all etcd members did not agree on the same leader. Therefore, I added `WaitLeader` function to the `Cluster` interface. But the `etcdserver: unhealthy cluster` error did not disappear. Finally I found it was caused by `--strict-reconfig-check`.